internal/pkg/launcher/rdp.go

```diff
@@ -0,0 +1,43 @@
+package launcher
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cloudradar-monitoring/rportcli/internal/pkg/exec"
+	"github.com/cloudradar-monitoring/rportcli/internal/pkg/models"
+	"github.com/cloudradar-monitoring/rportcli/internal/pkg/rdp"
+)
+
+type RDPFileWriter interface {
+	WriteRDPFile(fi models.FileInput) (filePath string, err error)
+}
+
+type RDPExecutor interface {
+	StartDefaultApp(filePath string) error
+}
```

Comment:
Are these interfaces used anywhere?

Reply:
@domenipavec They were used for the tests. But with the command recorder, there is no need to mock command.Execute by implementing an interface. I removed the interfaces.